feconf.py

```diff
@@ -37,10 +39,9 @@
 # or if you want to use minified resources in the development environment.
 
 if PLATFORM == 'gae':
-    DEV_MODE = (
-        (not os.environ.get('SERVER_SOFTWARE') or
-         os.environ['SERVER_SOFTWARE'].startswith('Development')) and
-        not FORCE_PROD_MODE)
+    if (constants.DEV_MODE and os.environ.get('SERVER_SOFTWARE') and
```

Comment:
Let's get rid of the PLATFORM 'gae' stuff, fix exception text, and remove FORCE_PROD_MODE above if not needed.